test/linalg/tridiag.jl

```diff
@@ -4,6 +4,35 @@ module TridiagTest
 using Base.Test
 debug = false
 
+# Test approximate equality of vectors or columns of matrices modulo floating
+# point roundoff and phase (sign) differences.
+#
+# This function is designed to test for equality between vectors of floating point
+# numbers when the vectors are defined only up to a global phase or sign, such as
+# normalized eigenvectors or singular vectors. The global phase is usually
+# defined consistently, but may occasionally change due to small differences in
+# floating point rounding noise or rounding modes, or through the use of
+# different conventions in different algorithms. As a result, most tests checking
+# such vectors have to detect and discard such overall phase differences.
+#
+# Inputs:
+#     a, b:: StridedVecOrMat to be compared
+#     err :: Default: m^3*(eps(S)+eps(T)), where m is the number of rows
+#
+# Raises an error if any columnwise vector norm exceeds err. Otherwise, returns
+# nothing.
+isdefined(:test_approx_eq_modphase) ||
+function test_approx_eq_modphase{S<:Real,T<:Real}(
+        a::StridedVecOrMat{S}, b::StridedVecOrMat{T}, err=nothing)
```

Comment:
Rather than copy-and-paste this into two files, can you just put it in a separate file and `include` it?